### ISSUE TITLE
Duplicate 'report_counts ' variable  

### ISSUE THREAD

**Issue by aeciopires** [voxpupuli/puppet-puppetboard#224]
The 'report_counts' variable is duplicate in lines 160 and 165 of manifest 'puppetboard/manifests/init.pp' in 4.1.0 version of module.

**Comment by bastelfreak** [comment]
Hi @aeciopires, thanks for bringing this up. Could you create a patch for this?

**Comment by aeciopires** [comment]
Hi @bastelfreak, the problem occurred in Debian 9, Ubuntu Server 18.04 and CentOS 7.

I looked the current code of manifest in Github and checked that duplicate was removed (https://github.com/voxpupuli/puppet-puppetboard/blob/master/manifests/init.pp). 

I tested the current code of module and worket in Debian 9, Ubuntu Server 18.04 and CentOS 7.

The solution is generate new version of module puppet-puppetboard at Puppet-Forge. :-)
